File: Server/domain/entities/global_card.go
package entities

import "time"

type GlobalCard struct {
	ID        int       `json:"id"`
	DeckID    int       `json:"deck_id"`
	Question  string    `json:"question"`
	Answer    string    `json:"answer"`
	AuthorID  int       `json:"author_id"`
	CreatedAt time.Time `json:"created_at"`
	UpdatedAt time.Time `json:"updated_at"`
	Version   int       `json:"version"`
}

File: lib/domain/entities/GlobalCard.dart
class GlobalCard {
  final int ID;
  final int deckID;
  final String question;
  final String answer;
  final int authorID;
  final DateTime createdAt;
  final DateTime updatedAt;
  final int version;
  GlobalCard ({
    required this.ID,
    required this.deckID,
    required this.question,
    required this.answer,
    required this.createdAt,
    required this.updatedAt,
    required this.authorID,
    required this.version,
  });
}